doc/tutorials/using_ubuntu_as_sos.rst

```diff
@@ -277,8 +317,8 @@ For the User OS, we are using the same `Clear Linux`_ release version as the Ser
 
   .. code-block:: none
 
-     -s 3,virtio-blk,/root/clear-24040-kvm.img
-     -k /lib/modules/kernel/org.clearlinux.pk414-standard.4.14.57-69
+     -s 3,virtio-blk,/root/clear-26440-kvm.img
```

Comment:
We should have a note here as the user is most likely not `root` and hence the file will be elsewhere on his filesystem. (using `~\` won't work here unless the user uses `sudo -E` to launch the script)

Reply:
@gvancuts does this need fixing before we merge and publish?